extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -246,6 +251,8 @@ function getDialog(service: IObjectManagementService, dialogOptions: ObjectManag
 			return new UserDialog(service, dialogOptions);
 		case ObjectManagement.NodeType.Database:
 			return new DatabaseDialog(service, dialogOptions);
+		case ObjectManagement.NodeType.DatabaseProperties:
```

Comment:
Since databases are going to be using a different dialog for new and edit I would suggest updating this function to take in a new `DialogType` value which specifies either `New`, `Edit` or `Properties`. For most of these nothing will change, then just for the Database entry have it change dialog based on what type it is. 

@alanrenmsft Any issues with that?

Reply:
I added an extra option to identify the command we are processing and used it to specify the respective dialog.